presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeScheduler.java

```diff
@@ -308,6 +310,10 @@ public static SplitPlacementResult selectDistributionNodes(
                     assignmentStats.getQueuedSplitCountForStage(node) < maxPendingSplitsPerTask) {
                 if (isCacheable) {
                     split = new Split(split.getConnectorId(), split.getTransactionHandle(), split.getConnectorSplit(), split.getLifespan(), new SplitContext(true));
+                    nodeSelectionStats.incrementBucketedPreferredNodeSelectedCount();
```

Comment:
Sorry if it is mentioned before, does bucketed table node selection have non-primary preferred node?

Reply:
Good question, actually we don't have that for bucketed table node selection; Just primary and non-preferred

Reply:
Is it necessary for us to separate bucketed from unbucketed? Can we merge them into existing stats?

Reply:
yeah, I also thought about that at first, then I feel it would be confusing since unbucketed situation has three status and bucketed has two; thus add those two for easy debugging when we have mixed situation